tests/production/DeleteUserSettingsFromCouchTests.js

```diff
@@ -237,8 +237,9 @@ gpii.tests.productionConfigTesting.deleteTestRecordsFromDatabaseTests = [{
         getOsGnomeKey: {
             type: "kettle.test.request.http",
             options: {
-                port: "5984",
-                hostname: "couchdb",
+                port: gpii.tests.productionConfigTesting.couchdbUrl.port,
```

Comment:
It's probably worthwhile to create a base grade with options at line 240-242 and have all test requests based off this base grade as these options are repeated for all request definitions.